test/evergreen/tcmalloc_install_or_build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Make a tcmalloc shared library avilable for preloading into the environment.
+
+set -u
+
+if [[ $# -ne 1 ]]; then
+    echo "Usage: $0 build-variant"
```

Comment:
nit: prefer to use `<build-variant>` to indicate the argument is not a fixed value.

Reply:
👍